client/src/app/page/add-task-form/add-task-form.component.html

```diff
@@ -44,22 +47,22 @@ <h5>Sorry, we have some problems</h5>
     </div>
     <div class="row form-row">
       <input type="text" name="taskSummary" formControlName="taskSummary" id="taskSummary" required minlength="9" maxlength="40" placeholder="Summary message" value="" />
-      <div *ngIf="taskSummary.invalid && (taskSummary.dirty || taskSummary.touched)" class="alert alert-danger">
-        <span *ngIf="taskSummary.errors.required">
+      <div *ngIf="taskSummary.invalid && (taskSummary.dirty || taskSummary.touched)" class="info-message">
+        <div *ngIf="taskSummary.errors.required" class="alert alert-danger visible-temporarily">
           Task name is required.
-        </span>
-        <span *ngIf="taskSummary.errors.minlength">
+        </div>
+        <div *ngIf="taskSummary.errors.minlength" class="alert alert-danger visible-temporarily">
           Task name must be at least 9 characters long.
-        </span>
-        <span *ngIf="taskSummary.errors.maxlength">
+        </div>
+        <div *ngIf="taskSummary.errors.maxlength" class="alert alert-danger visible-temporarily">
           Task name must be less 40 characters long.
-        </span>
+        </div>
       </div>
     </div>
     <div class="row form-row">
       <textarea name="taskDescription" formControlName="taskDescription" maxlength="400" id="taskDescription" class="textarea-field" placeholder="Task description"></textarea>
-      <div *ngIf="taskDescription.invalid && (taskDescription.dirty || taskDescription.touched)" class="alert alert-danger">
-        <span *ngIf="taskDescription.errors.maxlength">
+      <div *ngIf="taskDescription.invalid && (taskDescription.dirty || taskDescription.touched)" class="info-message">
```

Comment:
`taskDescription.invalid && (taskDescription.dirty || taskDescription.touched)` move it to JS file